### PR TITLE
Remove IRC

### PR DESCRIPTION
**Pull request by tristanweir**
Take out references to our now-unmonitored IRC channel. Convert all IRC nicknames to Slack usernames

### CODE REVIEW

**Comment by gdestuynder** [comment]
These channels are still active and open. If we remove them they should be closed first IMO. Why do you call this unmonitored?

File: docs/index.md [hunk 1]
```diff
@@ -22,20 +22,18 @@ Infosec assists Mozillians in defining and operating security controls to ensure
...
-  - Caglar Ulucenk *[:Cag]*
-  - Tristan Weir *[:weir]*
-    - Alicia Smith *[:phrozyn]*
-    - Brandon Myers *[:pwnbus]*
-    - Lucius Bono *[:lucius]*
-    - Michal Purzynski *[:michal`]*
-    - Emma Rose *[:emrose]*
+  - Guillaume Destuynder [🐦](https://twitter.com/kangsterizer) [💻](https://github.com/gdestuynder/) *[@kang]*
+  - Gene Wood *[@gene]*
+  - April King *[@April]*
```
Comment (floatingatoll) [inline docs/index.md]: Should be `@april` (lowercased)

File: docs/index.md [hunk 2]
```diff
@@ -22,20 +22,18 @@ Infosec assists Mozillians in defining and operating security controls to ensure
...
+  - Andrew Krug *[@andrew]*
+  - Justin Dow *[@jabba]*
+  - Jonathan Claudius *[@claudijd]*
+  - Caglar Ulucenk *[@Cag]*
```
Comment (floatingatoll) [inline docs/index.md]: Should be `@cag` (lowercased)

**Comment by gene1wood** [comment]
All IRC references have since been removed

**Pull request closed without merging**